README.md

```diff
@@ -36,7 +38,15 @@ npm run dev # starts the app, go to http://localhost:3000
 
 **Notes**
 
-As long as `npm run dev` is running, your changes will be visible on http://localhost:3000 live, no need to restart your server.
+As long as `npm run dev` is running, your changes will be visible on
+http://localhost:3000 live, no need to restart your server.
+
+## Guidance
+
+- Please do not modify the linting rules in your main PR. If you have
+  suggestions or opinions on how to improve DX please do so in a comment or
+  along side your submission. We review the changes against the main repository
```

Comment:
I suggest this rewording to emphasize that it's for practical reasons and not that we don't want them to have an opinion.
```suggestion
  along side your submission. This will make it easier for us to review your challenge as we will compare your changes against the main repository
```